projects/ngqp/core/src/lib/directives/query-param-group.service.ts

```diff
@@ -142,9 +142,9 @@ export class QueryParamGroupService implements OnDestroy {
     }
 
     /**
-     * Deregisters a {@link QueryParamNameDirective} directive by referencing its name.
+     * Unregisters a {@link QueryParamController} by referencing its name.
```

Comment:
I would actually like to retain the old name (deregister) as none is clearly better and the change is unrelated to the feature. I also somewhat prefer "deregister" because it has a stronger connotation of "it was registered before and now we remove that" :-)

Reply:
Ok